cmd/chart-repo/utils.go

```diff
@@ -41,8 +41,9 @@ import (
 )
 
 const (
-	chartCollection      = "charts"
-	chartFilesCollection = "files"
+	chartCollection       = "charts"
+	chartFilesCollection  = "files"
+	defaultTimeoutSeconds = 180
```

Comment:
180 is way too high here, we don't want to wait 2 minutes if an icon takes too long to fetch, it's better to skip it and process it next time. I suggest 30s, or keep it at 10s. This is not the same case as helm-crd, as this command runs every hour.

Reply:
fair enough

Reply:
reverted to 10s